pages/linux/pw-link.md

```diff
@@ -3,9 +3,9 @@
 > Manage links between ports in PipeWire.
 > More information: <https://gitlab.freedesktop.org/pipewire/pipewire/-/wikis/Virtual-Devices>.
 
-- List all audio output and input ports:
+- List all audio output and input ports with their ids:
```

Comment:
Welcome to tldr and thanks for your contribution, I would suggest capitalizing ID in the description.

```suggestion
- List all audio output and input ports with their IDs:
```